CONTRIBUTING.md

```diff
@@ -105,17 +103,22 @@ with what you'd like to do or have ideas that will help you do it.
 
 ### Adding a new library
 
-If you want to submit type stubs for a new library, you need to
+Stubs for new third party libraries are generally not accepted into
+typeshed anymore.  Such stubs should either be added directly to the
+third party package or distributed as a separate package.  See
+[PEP 561](https://www.python.org/dev/peps/pep-0561/) for details.
+
+When creating a separate package, you should
 **contact the maintainers of the original library** first to let them
 know and **get their permission**.  Do it by opening an issue on their
 project's bug tracker.  This gives them the opportunity to
 consider adopting type hints directly in their codebase (which you
-should prefer to external type stubs).  When the project owners agree
-for you to submit stubs here, open a pull request **referencing the
-message where you received permission**.
+should prefer to external type stubs).
 
-Make sure your changes pass the tests (the [README](README.md#running-the-tests)
-has more information).
+If you wish typeshed maintainers to review your third party stubs,
+please [open a new issue](https://github.com/python/typeshed/issues/new)
+in the typeshed repository with a link to the GitHub pull request
+you wish be have reviewed.
```

Comment:
This text asks for GitHub PRs for review requests. Should we broaden this or do you think it is okay to be this specific?

Reply:
I'd rather just say "to the code you would like us to review". No need to be overly specific. We should perhaps clarify though that there is no guarantee that we'll review their code quickly (or at all), since we're all volunteers.